tests/phpunit/tests/rest-api/rest-server.php

```diff
@@ -1119,6 +1119,46 @@ public function test_get_index() {
 		$this->assertArrayHasKey( 'site_icon_url', $data );
 	}
 
+	/**
+	 * @ticket 57902
+	 *
+	 * @covers WP_REST_Server::get_index
+	 */
+	public function test_get_index_fields_name() {
+		$server = new WP_REST_Server();
+
+		$request = new WP_REST_Request( 'GET', '/' );
+		$request->set_param( '_fields', 'name' );
+		$index = $server->dispatch( $request );
+		$index = rest_filter_response_fields( $index, $server, $request );
+		$data  = $index->get_data();
+		$links = $index->get_links();
+
+		$this->assertArrayHasKey( 'name', $data );
+		$this->assertArrayNotHasKey( 'help', $links );
+	}
+
+	/**
+	 * @ticket 57902
+	 *
+	 * @covers WP_REST_Server::get_index
+	 *
+	 * @dataProvider data_get_index_should_return_help_and_not_name
+	 */
+	public function test_get_index_fields_links( $field ) {
```

Comment:
```suggestion
	public function test_get_index_should_return_help_and_not_name( $field ) {
```
To match the data provider's name and reflect the test more accurately.